docs/source/_src_user_guide/state_propagation/propagation_setup/propagator_types.rst

```diff
@@ -1,128 +1,72 @@
-- When extracting the current state from a :class:`~tudatpy.numerical_simulation.environment.Body` object
+- To describe the initial conditions of an object when creating propagator settings
+  (typically using the :func:`~tudatpy.numerical_simulation.propagation_setup.propagator.translational` or
+  :func:`~tudatpy.numerical_simulation.propagation_setup.propagator.rotational` functions)
+- As an output from the numerical propagation, extracted from the
+:attr:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults.state_history` of the
+:class:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults` (see :ref:`propagation_results`)
+
+
+- When extracting the current state from a :class:`~tudatpy.numerical_simulation.environment.Body` object during the propagation
 
 Internally, Tudat uses the *conventional* state in the following places:
 
 - To update the environment model of an object (this also means that the states extracted from the body are expressed
   In the conventional coordinates);
-- Tso update the acceleration model of an object.
+- To update the acceleration model of an object.
+
 
+Propagated Elements
+-------------------
 
-Propagated Coordinates
--------------------------
+The *propagated* coordinates, on the other hand, are used *only* to describe the state in the actual differential equations
+being used. A list of options for translational and rotational dynamics is given in
+:class:`~tudatpy.numerical_simulation.propagation_setup.propagator.TranslationalPropagatorType` and
+:class:`~tudatpy.numerical_simulation.propagation_setup.propagator.RotationalPropagatorType`, respectively.
+The choice of formulation for translational and
+rotational motion is specified when calling the  :func:`~tudatpy.numerical_simulation.propagation_setup.propagator.translational` and
+:func:`~tudatpy.numerical_simulation.propagation_setup.propagator.rotational` functions, respectively.
 
-The *propagated* coordinates, on the other hand, are used to describe the state in the actual differential equations
-being used. Thus, you will find these elements here:
+In Tudat, you will find the propagated elements in the following place:
 
-- As an output to the ``unprocessed_state_history`` function of a ``Simulator`` object (such as the :class:`~tudatpy.numerical_simulation.SingleArcSimulator` class)
+- As an output from the numerical propagation, extracted from the
+:attr:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults.unprocessed_state_history` of the
+:class:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults` (see :ref:`propagation_results`)
 
 Internally, Tudat uses the *propagated* state in the following place:
 
-- As the input/output of the numerical integrator's state derivative funcion
+- As the input/output of the numerical integrator's state derivative function (see :ref:`single_propagation_evaluation:` for more details).
```

Comment:
This reference is not working

Reply:
Fixed (:ref:`single_propagation_evaluation:` -> :ref:`single_propagation_evaluation`)